server/controllers/postController.ts

```diff
@@ -1,24 +1,42 @@
 import { initManagers } from '../db/initDB';
+import { SortOption, FilterObject } from '../db/dbManager';
 
 const { postManager, userManager } = initManagers();
 
 async function postPosts(req, res, next) {
 	const refreshToken = req.cookies.refreshToken;
 	const username = await userManager.findRefreshToken(refreshToken);
-	const result = await postManager.addPost(req.body.title, req.body.body, username);
+	let result;
+	const { body: { title }, body: { body: postBody }, body: { parent } } = req;
```

Comment:
I'm not sure if I understand the nesting correctly in case of body, but I believe it could be destructed like this:

```suggestion
	const { body: { title, body, parent } } = req;
```